app/views/users/_edit_form.html.erb

```diff
@@ -69,7 +70,7 @@
         <!-- button for creating new users -->
         <button class="btn btn-lg btn-primary btn-save" type="submit" tabindex="7"><%= t('users._form.sign_up') %></button>
         <a class="btn btn-lg btn-default" href="/login"><%= t('users._form.log_in') %></a>
-      <% elsif params[:controller] == "users" && params[:action] == "edit" %>
+      <% elsif params[:controller] == "users" && (params[:action] == "edit" || params[:action] == "update") %>
```

Comment:
You don't need elsif condition here because there is no if condition.

Reply:
there is an if statement above it I just added "|| params[:action] == "update" " to make sure the save button appears once someone enters a wrong password confirmation or invalid email

Reply:
Those are not need from 69-73 line number.

Reply:
should I remove the if else condition and just keep the button html ?

Reply:
i.e remove line 69-73 and line 76

Reply:
Yes